src/components/SQForm/useFormButton.js

```diff
@@ -3,26 +3,59 @@ import {useFormikContext} from 'formik';
 import {useDebouncedCallback} from 'use-debounce';
 import {hasUpdated} from '../../utils';
 
-export function useFormButton(isDisabled, shouldRequireFieldUpdates, onClick) {
-  const {values, initialValues, isValid, ...rest} = useFormikContext();
+export const BUTTON_TYPES = {
+  SUBMIT: 'submit',
+  RESET: 'reset'
+};
+
+export function useFormButton({
+  isDisabled = false,
+  shouldRequireFieldUpdates = false,
+  onClick,
+  buttonType
+}) {
```

Comment:
I believe this is a breaking change. Consumers will need to switch their params to an object